### ISSUE TITLE
I got a different 3D plot, when I followed readme

### ISSUE THREAD

**Issue by hibiscuslee** [YiLab-SC/scHolography#15]
Hi,
As mentioned in the readme, I used thehuman skin scRNA-seq data and the  human skin ST 10X Visium data, and then I get the same dimplot image.
![fig1](https://github.com/user-attachments/assets/0ae93e86-3966-405c-b516-f6ee867333e4)

But after dataAlign and trainHolography, when I try to visualize the 3D structure colored by celltype, I got the different image：
![fig2](https://github.com/user-attachments/assets/009d8e9b-73a0-499c-9496-303b3677ae91)
When I try to rotate the view, I realized that it didn't match the picture
![2](https://github.com/user-attachments/assets/e90385ef-5546-44b3-b06c-60e2c552155c)

As I continued to try to draw the box plot and CompPlot, I found that there was also a big difference.
![fig3](https://github.com/user-attachments/assets/e67ad8d5-d098-4ed4-b09f-fbe2aec0d315)
![fig4_combined](https://github.com/user-attachments/assets/917d0bea-38cb-43c9-9daf-d6fd60e6a342)

`
neighbor.comp$significance
$Suprabasal
Suprabasal 
         0 

$Basal
       Basal   Suprabasal 
0.000000e+00 2.073902e-07 

$`Glandular Epithelium`
Glandular Epithelium 
                   0 

$Dermal
               Dermal           Endothelial Lymphatic Endothelial 
         0.000000e+00          5.809266e-15          1.633022e-17 
              Schwann         Smooth Muscle 
         2.188646e-15          3.346468e-44 

$Endothelial
          Endothelial Lymphatic Endothelial               Schwann 
         0.000000e+00          2.728696e-03          4.777623e-11 
        Smooth Muscle 
         2.314087e-04 

$`Lymphatic Endothelial`
                Basal                Dermal           Endothelial 
         2.677175e-15          7.811286e-05          3.912714e-15 
 Glandular Epithelium                Immune Lymphatic Endothelial 
         1.345728e-09          1.735365e-21         1.033488e-154 
              Schwann 
         9.794988e-06 

$`Smooth Muscle`
Smooth Muscle 
2.495296e-207 

$Schwann
          Endothelial Lymphatic Endothelial               Schwann 
         1.106642e-02          1.026428e-02         1.195355e-265 

$Immune
                Basal                Immune Lymphatic Endothelial 
         7.820362e-08         3.229143e-146          1.727432e-16 
           Melanocyte 
         7.940980e-04 

$Melanocyte
                Basal           Endothelial                Immune 
         5.980825e-03          5.875973e-03          2.546411e-16 
Lymphatic Endothelial            Melanocyte            Suprabasal 
         2.519738e-02          6.656723e-82          1.273828e-02 `

It looks like the celltypes have different spatial distance. Is this because of the difference in training during the trainHolography?

> sessionInfo()
R version 4.2.1 (2022-06-23)
Platform: x86_64-conda-linux-gnu (64-bit)
Running under: CentOS Linux 7 (Core)

Matrix products: default
BLAS/LAPACK: /miniconda3/envs/scholography/lib/libopenblasp-r0.3.27.so

locale:
 [1] LC_CTYPE=zh_CN.UTF-8       LC_NUMERIC=C              
 [3] LC_TIME=zh_CN.UTF-8        LC_COLLATE=zh_CN.UTF-8    
 [5] LC_MONETARY=zh_CN.UTF-8    LC_MESSAGES=zh_CN.UTF-8   
 [7] LC_PAPER=zh_CN.UTF-8       LC_NAME=C                 
 [9] LC_ADDRESS=C               LC_TELEPHONE=C            
[11] LC_MEASUREMENT=zh_CN.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
 [1] future_1.32.0      viridis_0.6.5      viridisLite_0.4.2  RColorBrewer_1.1-3
 [5] dplyr_1.1.4        ggplot2_3.4.2      SeuratObject_4.1.3 Seurat_4.3.0      
 [9] scHolography_0.1.0 tensorflow_2.11.0  reticulate_1.39.0 

loaded via a namespace (and not attached):
  [1] Rtsne_0.17             colorspace_2.1-1       deldir_2.0-4          
  [4] ggridges_0.5.4         base64enc_0.1-3        spatstat.data_3.1-2   
  [7] farver_2.1.2           matchingR_1.3.3        leiden_0.4.3.1        
 [10] listenv_0.9.1          ggrepel_0.9.3          fansi_1.0.6           
 [13] codetools_0.2-20       splines_4.2.1          knitr_1.47            
 [16] zeallot_0.1.0          polyclip_1.10-4        jsonlite_1.8.8        
 [19] ica_1.0-3              cluster_2.1.6          png_0.1-8             
 [22] tfruns_1.5.3           uwot_0.1.14            shiny_1.8.1.1         
 [25] sctransform_0.3.5      spatstat.sparse_3.1-0  compiler_4.2.1        
 [28] httr_1.4.7             Matrix_1.5-4           fastmap_1.2.0         
 [31] lazyeval_0.2.2         cli_3.6.3              later_1.3.2           
 [34] htmltools_0.5.8.1      tools_4.2.1            igraph_2.0.3          
 [37] gtable_0.3.5           glue_1.7.0             RANN_2.6.2            
 [40] reshape2_1.4.4         rappdirs_0.3.3         gmodels_2.19.1        
 [43] Rcpp_1.0.12            scattermore_0.8        spatstat.univar_3.0-1 
 [46] vctrs_0.6.5            gdata_3.0.0            spatstat.explore_3.1-0
 [49] nlme_3.1-166           progressr_0.14.0       crosstalk_1.2.1       
 [52] lmtest_0.9-40          spatstat.random_3.3-1  xfun_0.45             
 [55] stringr_1.5.1          globals_0.16.3         mime_0.12             
 [58] miniUI_0.1.1.1         lifecycle_1.0.4        irlba_2.3.5.1         
 [61] gtools_3.9.5           goftest_1.2-3          MASS_7.3-59           
 [64] zoo_1.8-12             scales_1.3.0           promises_1.3.0        
 [67] spatstat.utils_3.1-0   parallel_4.2.1         yaml_2.3.8            
 [70] pbapply_1.7-2          gridExtra_2.3          keras_2.11.1          
 [73] stringi_1.7.12         rlang_1.1.4            pkgconfig_2.0.3       
 [76] matrixStats_1.4.1      evaluate_0.24.0        pracma_2.4.4          
 [79] lattice_0.22-6         ROCR_1.0-11            purrr_1.0.2           
 [82] tensor_1.5             patchwork_1.1.2        htmlwidgets_1.6.4     
 [85] cowplot_1.1.1          tidyselect_1.2.1       parallelly_1.38.0     
 [88] RcppAnnoy_0.0.20       plyr_1.8.9             magrittr_2.0.3        
 [91] R6_2.5.1               generics_0.1.3         pillar_1.9.0          
 [94] whisker_0.4.1          withr_3.0.0            fitdistrplus_1.1-11   
 [97] survival_3.7-0         abind_1.4-5            sp_2.1-4              
[100] tibble_3.2.1           future.apply_1.11.2    KernSmooth_2.23-24    
[103] utf8_1.2.4             spatstat.geom_3.1-0    plotly_4.10.1         
[106] rmarkdown_2.27         grid_4.2.1             data.table_1.16.0     
[109] digest_0.6.36          xtable_1.8-4           tidyr_1.3.1           
[112] httpuv_1.6.15          munsell_0.5.1

**Comment by cadyyuheng** [comment]
Hi @hibiscuslee , Thanks for your interest in scHolography, and congratulations on successfully completing the tutorials! The differences you observed are entirely normal. The exact 3D visualizations and cell type SMN variations can differ slightly across devices due to the handling of random numbers during training. While we set an internal seed to eliminate variation across runs on the same device or machine, as you've noticed, slight differences can still occur on different platforms than the one used to test the tutorial.
